static-checks/ansible/syntax-check/test.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+import subprocess
+
+from lib import ansible, util, oscap, results
+
+
+ansible.install_deps()
+
+ds = oscap.global_ds()
+all_profiles = ds.profiles.keys()
+
+# Generated playbooks
+for profile in all_profiles:
```

Comment:
You should be able to just do
```python
for profile in ds.profiles:
```
as `iter(my_dict)` is the same as `iter(my_dict.keys())`.